source/System.h

```diff
@@ -102,6 +102,9 @@ class System {
 	double ExtraHyperArrivalDistance() const;
 	// Additional travel distance to target for ships entering using a jumpdrive.
 	double ExtraJumpArrivalDistance() const;
+
+	double DepartureDistance() const;
```

Comment:
It would probably make sense for this to consider JD and Hyper differently like it happens with the arrival distance? Not sure what is going on with that exactly though

Reply:
I can do so

Reply:
We actually have [an RFC in progress](https://github.com/EndlessSkyCommunity/endless-sky/pull/51/files) where syntax for jump and hyperdrives were discussed. But don't worry about RFCs in progress; they are not official directions until they are officially merged.

Reply:
Hmmm... I can try to implement some of the things there, I dont see the use of a max though

Reply:
Please do keep things small and simple and just focus on basic departure distance in this PR. Differentiating between hyperdrives and jumpdrives can also be done in later PRs. (And just refer to the RFC for questions on how that can be done later.)

Reply:
sorry, too late